packages/node-cli/src/mappers/CreateProjectActionResultMapper.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+const ActionResult = require('../commands/actionresult/ActionResult');
+const CreateProjectActionResult = require('../commands/actionresult/CreateProjectActionResult');
+const DefaultActionResultMapper = require('./DefaultActionResultMapper');
+
+
+class CreateProjectActionResultMapper extends DefaultActionResultMapper {
+    constructor() {
+        super()
+    }
+
+    createActionResultFrom(actionCreateProjectData, projectType, includeUnitTesting) {
+        if (actionCreateProjectData.operationResult.status === ActionResult.SUCCESS) {
+            return CreateProjectActionResult.Builder.withSuccess(
+                actionCreateProjectData.operationResult.data,
+                actionCreateProjectData.operationResult.resultMessage,
+                projectType,
+                actionCreateProjectData.projectDirectory,
+                includeUnitTesting,
+                actionCreateProjectData.npmInstallSuccess
```

Comment:
The objective of the Builder patter is exactly to not have this. Instead, we should have something like:
```
CreateProjectActionResult.Builder
  .withData(actionCreateProjectData.operationResult.data)
  .withResultMessage(actionCreateProjectData.operationResult.resultMessage)
  .withType(projectType)
  .fromDirectry(actionCreateProjectData.projectDirectory)
  .withUnitTesting()
  .withSuccessfullNpmInstalled()
```

Reply:
Yeah1 True... I don't know how I can have don this :S